client/src/modules/Schedule/components/TableView/TableView.tsx

```diff
@@ -47,7 +51,8 @@ const getColumns = ({
           </a>
         );
       },
-      ...getColumnSearchProps('name'),
+      filteredValue: filteredInfo.name || null,
+      ...getColumnSearchProps('name', undefined),
```

Comment:
Perhaps, we don't need to put here `undefined` explicitly :)

Reply:
Sure, my bad. Fixed